### DISCUSSION TITLE
Sidebar only stashable on small screens

### DISCUSSION THREAD

**Discussion by brnhache** [livewire/flux#362] [Feature Requests]
I can hide the sidebar with javascript but id prefer to do it in a Flux way. Is this possible on medium and large screens?

**Comment by RiadhKHEDHIRI** [comment]
I just came across this. Is this related?

![image](https://github.com/user-attachments/assets/183ddb0f-f9ab-40ea-86eb-85edddc3b6b4)

**Reply by brnhache** [reply]
it looks like your tailwind might not be configured properly. i dont have any errors in the console like that

**Comment by ruskiyos** [comment]
@brnhache I haven't been able to come up with a clean way to stash on medium/large screens. Do you mind sharing how you got that to work?

I don't understand why there would be a restriction as there are many cases where having a collapsible side-bar that isn't restricted to mobile view is useful. Google maps (for example) loads with a stashed menu. 
![image](https://github.com/user-attachments/assets/285a39e0-5228-4cc4-a803-8cb943ee9f8f)

Not sure who we need to ping to get the attention of flux developers, but I hope they can add a way to control that.

**Reply by brnhache** [reply]
replying in [#877](https://github.com/livewire/flux/discussions/877)

**Comment by joshhanley** [comment]
Hey everyone, looks like we have a duplicate feature request between this one and #877. Even though this one is older, #877 has more upvotes and support, so I'm going to close this one in favour of that one. If you could all add your support to #877 by upvoting it and adding a comment there, that will help us prioritise future features. Thanks!
